core/src/Kokkos_Pair.hpp

```diff
@@ -413,12 +413,13 @@ KOKKOS_FORCEINLINE_FUNCTION pair<T1&, T2&> tie(T1& x, T2& y) {
   return (pair<T1&, T2&>(x, y));
 }
 
+#ifndef KOKKOS_ENABLE_DEPRECATED_CODE_4
```

Comment:
Should it be `#ifdef`? I'm getting build failures when `KOKKOS_ENABLE_DEPRECATED_CODE_4` is OFF

Reply:
Looks like it.